resources/Audience_id.csv

```diff
@@ -0,0 +1,247 @@
+aud_id,label
```

Comment:
Hmmm. I was  thinking it would just be kept in a common general area like '/resources'. 

We can expand the common area with additional folders as needed. For example, in time would move this file to something like 'resources/db'.